css/components/dashboards/dashboards-subheader-block.scss

```diff
@@ -0,0 +1,7 @@
+.c-dashboards-subheader-block {
+  padding-bottom: 45px;
+  padding-top: 70px;
```

Comment:
@dancer2090 Better to use for paddings, margins, etc... a factor of the variable `$space`, e.g. `$space * 14` in the case of this line

Reply:
Fixed https://github.com/resource-watch/resource-watch/pull/1179/commits/b044ec7607c66fcb8d92500e2fa4f7d7d57dec2d